.github/workflows/localization-update.yml

```diff
@@ -0,0 +1,29 @@
+# Transform localizations from TranslateWiki.net
+#
+# As of late 2020, TranslateWiki creates PRs automatically, in line with
+# how they work with other repos. However, the strings still need to be
+# translated to work well with the iOS app. This script runs when a new PR
+# is created by TranslateWiki, to add everything the app needs to the PR.
+#
+name: Update localizations from TranslateWiki
+
+on:
+  pull_request:
```

Comment:
I'd suggest running this on `push` instead of `pull_request` to handle cases where the localization script adds new localizations after the PR is already opened